test/e2e/allocation.cancel.test.js

```diff
@@ -0,0 +1,26 @@
+import { pmuUser } from './_roles'
+import { allocations, newAllocation } from './_routes'
+import { allocationJourney } from './pages/'
+
+fixture('Cancel allocation').beforeEach(async t => {
+  await t.useRole(pmuUser).navigateTo(newAllocation)
+  await allocationJourney.createAllocation()
+  await t.navigateTo(allocations)
+})
+test('Cancel allocation', async t => {
+  await t.click(
+    allocationJourney.allocationCancelPage.nodes.linkToFirstAllocation
+  )
+  await allocationJourney.scrollToBottom()
```

Comment:
Maybe this and `getCurrentUrl` could be called using `page.scrollToBottom()` and `page.getCurrentUrl()` to make it more clear which page object those methods actually exist on.

Reply:
I understand your argument. The reason why I haven't done it is because I am not too keen on importing both parent and child objects in order to consume methods that exist on the parent and are not overridden in the child. It's like throwing a tiny curveball to those who read the codebase, for I act there like there was a specific reason (overridden method? no inheritance at all?) to not consume everything from the child. Makes sense?